packages/components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -22,19 +54,47 @@ const meta: Meta = {
 } as Meta;
 export default meta;
 
-export const Tooltip: StoryObj<TooltipProps> = {
-    render: args => (
+export const Component = (args: TooltipProps) => {
```

Comment:
Isn't it too generic? I know it's just a storybook, but it's exported so I don't know 🤷‍♂️

Reply:
Fixed in ab26c5f007